include/swift/AST/DiagnosticsSema.def

```diff
@@ -1005,6 +1005,10 @@ ERROR(module_not_testable,Fatal,
 ERROR(module_not_compiled_for_private_import,none,
       "module %0 was not compiled for private import", (Identifier))
 
+ERROR(in_package_module_not_compiled_from_source,Fatal,
+      "module %0 is in package %1 and can only be loaded if built from source",
+      (Identifier, Identifier))
```

Comment:
Can we have a mention of "module interface" in this diagnostics? It would point to the problem, we usually try to describe both the problem and the solution in error messages.

Also, could we show the path to the swiftinterface? In practice, it would likely show when a module is loaded from the SDK instead of the local build directory like expected. It may be enough for people to figure out how to fix this issue without reaching out to compiler engineers.

Reply:
sounds good, will do